ceph.spec.in

```diff
@@ -512,7 +512,7 @@ Group:          System/Filesystems
 %endif
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
 Requires:       python%{python3_pkgversion}-numpy
-%if 0%{?fedora} || 0%{?suse_version}
+%if 0%{?fedora} || ( 0%{?suse_version} && 0%{?is_opensuse} )
```

Comment:
```suggestion
%if 0%{?fedora} || 0%{?is_opensuse}
```

i think `is_opensuse` alone would suffice. see https://en.opensuse.org/openSUSE:Packaging_for_Leap#RPM_Distro_Version_Macros

on second thought, shall we disable this package from being generated on distros where python3-scikit-learn is missing? as, IMHO, the existence of the package is a misleading signal to users of those distros.

Reply:
good points, @tchaikov - please do another review round?